code/datums/brain_damage/hypnosis.dm

```diff
@@ -25,14 +25,14 @@
 /datum/brain_trauma/hypnosis/on_gain()
 	message_admins("[ADMIN_LOOKUPFLW(owner)] was hypnotized with the phrase '[hypnotic_phrase]'.")
 	log_game("[key_name(owner)] was hypnotized with the phrase '[hypnotic_phrase]'.")
-	to_chat(owner, "<span class='reallybig hypnophrase'>[hypnotic_phrase]</span>")
-	to_chat(owner, "<span class='notice'>[pick("You feel your thoughts focusing on this phrase... you can't seem to get it out of your head.",\
+	to_chat(owner, span_hypnophrase("[hypnotic_phrase]"))
```

Comment:
This hypnophrase won't be bigtext like the original span did

Reply:
I was wondering about this. Perhaps I'll make a hypnophraselarge helper for it